co-simulation tool/README.md

```diff
@@ -0,0 +1,248 @@
+ | └─ <scenarioName>.sumocfg ............ SUMO configuration file
+ └─ scenario_config.json ................ Basic configuration of the simulation scenario
+```
+
+Here the scenario configuration file , the CARLA configuration file and bridge file are discussed. For other configuration files and scenario creation, refer to the documentation of Eclipse MOSAIC [[link](https://www.eclipse.org/mosaic/docs/)]. Moreover, the  CARLA 3D maps corresponding to the scenarios must be generated for co-simulation scenarios.
+
+**a.**   **Scenario configuration file**
+
+The scenario_config.json is the main configuration file of a co-simulation scenario. A co-simulation scenario file looks like the following example.
+
+```json
+{
+  "simulation": {
+   "id": "town04",
+   "duration": "200s",
+   "randomSeed": 212323853,
+   "projection": {
+      "centerCoordinates": {
+        	"latitude": 52.63,
+        	"longitude": 13.56
+      },
+      "cartesianOffset": {
+       		"x": -563984.16,
+      	 	"y": -5933566.87
+     }
+    },
+   "network": {
+    	"netMask": "255.255.0.0",
+    	"vehicleNet": "10.1.0.0",
+    	"rsuNet": "10.2.0.0",
+    	"tlNet": "10.3.0.0",
+   	 	"csNet": "10.4.0.0",
+    	"serverNet": "10.5.0.0",
+    	"tmcNet": "10.6.0.0"
+    }
+  },
+  "federates": {
+  		"application": true,
+  		"cell": false,
+  		"environment": false,
+  		"sns": false,
+  		"ns3": false,
+  		"omnetpp": false,
+  		"output": true,
+  		"carla": true,
+  		"sumo": true
+  }
+}
+```
+
+The federate tags show which simulators are  used in the simulation. For the co-simulation scenarios, CARLA and SUMO federate must be enabled. If the CARLA federate is disabled, the simulation of SUMO and MOSAIC will be run.
+
+**b.**   **Carla configuration file**
+
+The CARLA ambassador can be configured with the carla configuration file. The specific path is <scenarioName>/carla/carla_config.json.  The example configuration files of CARLA ambassador on Linux systems and on Windows systems are shown in the following, respectively.
+
+**Linux**
+
+```json
+{
+    "updateInterval": 100,
+	"carlaUE4Path": "/CARLA_0.9.10/",
+	"bridgePath": "./scenarios/Town04_10/carla; bridge.sh",
+	"carlaConnectionPort": 8913
+}
+```
+
+**Windows**
+
+```json
+{
+    "updateInterval": 100,
+	"carlaUE4Path": "D:/CARLA_0.9.10/",
+	"bridgePath": "./scenarios/Town04_10/carla; bridge.bat",
+	"carlaConnectionPort": 8913
+}
+```
+
+The following table descripts the parameters in the CARLA configuration file.
+
+| Parameters          | **Type** | **Description**                                              | **Required** |
+| ------------------- | -------- | ------------------------------------------------------------ | ------------ |
+| updateInterval      | number   | The Interval after which the CARLA is updated.  The default unit is ms. Define the size  of one simulation step in CARLA. The updateInterval of Carla and Sumo  ambassador must be the same. | Yes          |
```

Comment:
Should indicate the minimal value of updateInterval is 100 and the default setting of updateInterval being predefined in the java code is 1000 for both carla and sumo configuration

Reply:
A note has been added in the description of updateInterval to indicate the minimal value and predefined value of updateInterval.